smoke_tests/tests/authenticated/azure/managed_apps_tests.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from tests.smoke_test_case import SmokeTestCase
+
+
+class ManagedAppsTests(SmokeTestCase):
+    AZURE_SUBSCRIPTION_ID = None
+
+    @staticmethod
+    def managed_apps_url() -> str:
+        return SmokeTestCase.build_bpm_url("/api/azure/v1/managedApps")
+
+    def test_retrieving_managed_apps(self):
+        params = {"azureSubscriptionId": self.AZURE_SUBSCRIPTION_ID}
```

Comment:
Should we assert first that `self.AZURE_SUBSCRIPTION_ID` is not None?

Reply:
And possibly `USER_TOKEN`?

Reply:
I figured we were doing that when setting up the test case.